### PR TITLE
Quick fix for communicators

### PR DESCRIPTION
**Pull request by Arokha**
This should be improved upon. Maybe even switch them to NTnet. It'd be nice to not have 4 different systems (pda messages, telecomms, exonet, ntnet, no offense neerti)

### CODE REVIEW

**Comment by Atermonera** [comment]
I wrote the commcard stuff for communicators with the intent of those phasing out PDAs, but mechoid's demanded messaging server-esque logs and impersonation, and I haven't worked on that project for a year or so now.

**Comment by Arokha** [comment]
Fixed conflict

File: code/game/machinery/telecomms/telecomunications.dm [hunk 1]
```diff
@@ -1,740 +1,712 @@
...
+
+	var/machinetype = 0 // just a hacky way of preventing alike machines from pairing
+	var/toggled = 1 	// Is it toggled on
+	var/on = 1
+	var/integrity = 100 // basically HP, loses integrity by heat
+	var/produces_heat = 1	//whether the machine will produce heat when on.
+	var/delay = 10 // how many process() ticks to delay per heat
+	var/long_range_link = 0	// Can you link it across Z levels or on the otherside of the 
+  
+  ? (Relay & Hub)
```
Comment (Atermonera) [inline code/game/machinery/telecomms/telecomunications.dm:36]: travis

**Comment by Arokha** [comment]
Just redid the whole branch because mlem.

**Pull request merged**